project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.5.4
+sbt.version=1.5.5
```

Comment:
This brings in some BSP updates (see https://github.com/sbt/sbt/releases/tag/v1.5.5), so I'm thinking it's probably worth merging!